README.adoc

```diff
@@ -6,134 +6,23 @@ image:https://travis-ci.com/devonfw/sonar-devon4j-plugin.svg?branch=master["Buil
 
 Plugin for https://sonarqube.org[SonarQube] to validate https://github.com/devonfw/devon4j/blob/develop/documentation/coding-conventions.asciidoc#packages[devon4j architecture].
 
-== Motivation
+== Documentation
 
-With https://github.com/devonfw/devon4j/[devon4j] you can build business applications very efficiently following elaborated https://github.com/devonfw/devon4j/wiki[guidelines and best practices].
-This includes a profound https://github.com/devonfw/devon4j/blob/develop/documentation/architecture.asciidoc#architecture[architecture blueprint] that is mapped to the code via clear https://github.com/devonfw/devon4j/blob/develop/documentation/coding-conventions.asciidoc#packages[packaging conventions].
+* link:documentation/motivation.adoc[Motivation]
+* link:documentation/installation.adoc[Installation]
+* link:documentation/configuration.adoc[Configuration]
+* link:documentation/qualityprofile.adoc[devonfw Java Quality Profile]
 
-This `sonar-devon4j-plugin` provides a plugin extending https://sonarqube.org[SonarQube] with the ability to validate your Java code according to the devon4j architecture.
+== Architectural Dependency Rules
 
-== Installation
+* link:documentation/rules-architecture.adoc[Architecture Rules]
```

Comment:
The level for this bullet is fine, but I would indent the next three bullet points one further using `**` instead of `*`. So that the chapters "Layer", "Component" & "Scope" are subchapters of "Architectural Dependency Rules".